src/muse/constraints.py

```diff
@@ -357,21 +357,26 @@ def max_capacity_expansion(
     if regions is not None and "region" in technologies.dims:
         techs = techs.sel(region=regions)
 
+    # Existing and forecasted capacity
+    initial = capacity.sel(year=year, drop=True)
+    forecasted = capacity.sel(year=forecast_year, drop=True)
+
+    # Max capacity addition constraint
     add_cap = techs.max_capacity_addition * forecast
 
+    # Total capacity limit constraint
     limit = techs.total_capacity_limit
-    forecasted = capacity.sel(year=forecast_year, drop=True)
-    total_cap = (limit - forecasted).clip(min=0).rename("total_cap")
+    total_cap = (limit - forecasted).clip(min=0)
 
+    # Max capacity growth constraint
     max_growth = techs.max_capacity_growth
-    initial = capacity.sel(year=year, drop=True)
+    growth_cap = initial * ((max_growth + 1) ** forecast) - forecasted
```

Comment:
Python might have other ideas, but in principle, mathematically speaking, these parenthesis are redundant.
```suggestion
    growth_cap = initial * (max_growth + 1) ** forecast - forecasted
```